electron/main.js

```diff
@@ -26,24 +39,52 @@ function createWindow() {
   );
 
   // Open the DevTools.
-  win.webContents.openDevTools();
-
+  if (isDevMode) {
+    win.webContents.openDevTools();
+  }
   // Emitted when the window is closed.
   win.on('closed', () => {
     // Dereference the window object, usually you would store windows
     // in an array if your app supports multi windows, this is the time
     // when you should delete the corresponding element.
     win = null;
   });
+
+  win.webContents.on('did-finish-load', () => {
+    // Windows: Handle deeplink url
+    if (process.platform == 'win32') {
+      // Keep only command line / deep linked arguments
+      deeplinkingUrl = process.argv.slice(1);
+    }
+    if (deeplinkingUrl) {
+      setTimeout(() => {
+        win.webContents.send('open-url-event', deeplinkingUrl);
+      }, 1000);
+    }
+  });
 }
 
+app.setAsDefaultProtocolClient('copay');
+app.setAsDefaultProtocolClient('bitpay');
+app.setAsDefaultProtocolClient('bitcoin');
 app.setVersion(appConfig.version);
 app.setName(appConfig.nameCase);
 
-let nameFolder = appConfig.name + '-v4';
+const getHomeDirPath = platform => {
+  switch (platform) {
+    case 'win32':
+      return process.env['USERPROFILE'];
```

Comment:
if the user change "userprofile" variable, the will no be able to find his wallets, right?

Reply:
can't we use a `homedir()` thing? if not, add this to the README or something

Reply:
Yes, right